src/Payment/PaymentTests/Suites.cs

```diff
@@ -0,0 +1,94 @@
+			{ GetValuesSuites.ThreeValues, new() { Cart = new() { CartItems = suiteCartItemsMap[GetValuesSuites.ThreeValues]}} },
+			{ GetValuesSuites.ManyValues, new() { Cart = new() { CartItems = suiteCartItemsMap[GetValuesSuites.ManyValues]}} },
+		};
+
+		/// <param name="seatStatus">
+		/// null means that <see cref="EventSeat"/>.Status would be a
+		/// random value of <see cref="SeatStatus"/> lower case string.
+		/// </param>
+		public static Payment GetPayment(GetValuesSuites suite, SeatStatus? seatStatus)
+		{
+			var payment = suitePaymentMap[suite];
+			payment.Status = PaymentStatus.InProgress.ToString().ToLowerInvariant();
+			// In true unit case for random values I would also add all possible values manually without random
+			// random can be brutal / unit test should be repeatable!
+			foreach (var item in payment.Cart.CartItems)
+			{
+				item.EventSeat.Status = seatStatus switch
+				{
+					SeatStatus.Booked => BookedStatus,
+					SeatStatus.Available => AvailableStatus,
+					SeatStatus.Sold => SoldStatus,
+					null => new Random().Next(0, 3) switch { 0 => BookedStatus, 1 => AvailableStatus, _ => SoldStatus },
+					_ => throw new NotImplementedException(nameof(seatStatus))
+				};
+			}
+
+			return payment;
+		}
+
+		private static Dictionary<GetValuesSuites, IList<CartItem>> suiteCartItemsMap => new()
+		{
+			{ GetValuesSuites.OneValue, [new() { EventSeat = new()  }] },
+			{ GetValuesSuites.ThreeValues, [new() { EventSeat = new()  }, new() { EventSeat = new()  }, new() { EventSeat = new()  }] },
+			{ GetValuesSuites.ManyValues, [
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+					new() { EventSeat = new()  },
+				] }
```

Comment:
))) how you managed to stop copy-pasting at this row?

Reply:
Hand has tired)